app/sample/src/main/res/values/styles.xml

```diff
@@ -1,11 +1,36 @@
 <resources>
 
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
-        <!-- Customize your theme here. -->
-        <item name="colorPrimary">@color/colorPrimary</item>
-        <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
-        <item name="colorAccent">@color/colorAccent</item>
+    <style name="AppTheme" parent="Theme.Sygic.Light.DarkActionBar">
+        <!--Todo: sample APP-->
+        <!--<item name="compassStyle">@style/TestCompassStyle</item>-->
+        <!--<item name="positionLockFabStyle">@style/TestPositionLockFabStyle</item>-->
+        <!--<item name="zoomControlsMenuStyle">@style/TestZoomControlsMenuStyle</item>-->
+    </style>
+
+    <style name="AppTheme.NoActionBar">
+        <item name="windowActionBar">false</item>
+        <item name="windowNoTitle">true</item>
+        <item name="android:statusBarColor">@android:color/transparent</item>
+    </style>
+
+    <style name="AppTheme.AppBarOverlay" parent="ThemeOverlay.AppCompat.Dark.ActionBar" />
+    <style name="AppTheme.PopupOverlay" parent="ThemeOverlay.AppCompat.Light" />
+
+    <!--Todo: sample APP-->
+    <style name="TestCompassStyle">
```

Comment:
preco _test_xyz?

Reply:
To bylo jen tak pro mě, je to momentálně nepoužité a připravené na tu sample APP, proto je tam to todo. Ale přejmenuji ;)